tests/e2e/utils/flows.js

```diff
@@ -297,6 +298,46 @@ export const shopperWCP = {
 		await shopperWCP.goToProductPageBySlug( productSlug );
 		await shopper.addToCart();
 	},
+
+	waitForErrorBanner: async (
+		errorText,
+		noticeSelector,
+		oldNoticeSelector
+	) => {
+		const errorBannerToCheck = ( async () => {
+			await expect( page ).toMatchElement( noticeSelector, {
+				text: errorText,
+			} );
+		} )();
+
+		const oldErrorBannerToCheck = ( async () => {
+			await expect( page ).toMatchElement( oldNoticeSelector, {
+				text: errorText,
+			} );
+		} )();
+
+		await Promise.race( [ errorBannerToCheck, oldErrorBannerToCheck ] );
```

Comment:
This allows us to choose from two promises that check old & new selectors. This way, we're able to support both selectors from the current and old WooCommerce version.